OpenRA.Mods.Common/UpdateRules/Rules/SplitTurretAimAnimation.cs

```diff
@@ -26,33 +27,54 @@ public override string Description
 			}
 		}
 
+		readonly List<Tuple<string, string>> locations = new List<Tuple<string, string>>();
+
+		public override IEnumerable<string> AfterUpdate(ModData modData)
+		{
+			var message = "AimSequence and ReloadPrefix have been split from WithTurretAttackAnimation\n"
+				+ "to the new WithTurretAimAnimation trait. The following location has been updated,\n"
+				+ "you may want to check the result for any necessary manual adjustments:\n"
+				+ UpdateUtils.FormatMessageList(locations.Select(n => n.Item1 + " (" + n.Item2 + ")"));
+
+			if (locations.Any())
+				yield return message;
```

Comment:
This doesn't seem to work.  If I revert the yaml change from #14846 and run the update rule it splits off `WithTurretAimAnimation` but does not report any locations.